cns/requestcontroller/kubecontroller/crdtranslator.go

```diff
@@ -36,7 +36,7 @@ func CRDStatusToNCRequest(crdStatus nnc.NodeNetworkConfigStatus) (cns.CreateNetw
 		ncRequest.SecondaryIPConfigs = make(map[string]cns.SecondaryIPConfig)
 		ncRequest.NetworkContainerid = nc.ID
 		ncRequest.NetworkContainerType = cns.Docker
-		ncRequest.Version = nc.Version
+		ncRequest.NCVersion = nc.Version
```

Comment:
Why is this changed? It should be set to ncRequest.Version only

Reply:
Will change it back to ncRequest.Version. 
I didn't reuse the Version since the usage of Version is not clear in CNS previously. There are Version, which compared with VMVersion and HostVersion which record NMA version. After double checking code, resusing Version should be fine.